book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -581,13 +582,13 @@ All supervised learning measures compare the difference between predicted values
 `mlr3` simplifies the process of bringing these quantities together by storing the predictions and true outcomes in the `r ref("Prediction", index = TRUE)` object as we have already seen.
 
 ```{r}
-prediction
+predictions
```

Comment:
Marc changed predictions -> prediction everywhere as part of unifying book terminology. I'm not saying change back but this chapter will now be out of sync with others

Reply:
I find the name `predictions` more intuitive as there's clearly more than one of them, but I'll leave it up to you whether to change it again.

Reply:
@be-marc how difficult would it be to update this everywhere? / how much time would it take?